colour/models/din99.py

```diff
@@ -28,17 +29,31 @@
 __email__ = 'colour-developers@colour-science.org'
 __status__ = 'Production'
 
-__all__ = ['Lab_to_DIN99', 'DIN99_to_Lab']
+__all__ = ['Lab_to_DIN99', 'DIN99_to_Lab', 'DIN99_METHODS']
 
+DIN99_METHODS = CaseInsensitiveMapping({
+    'ASTMD2244-07':
```

Comment:
Let's also make a `DIN99` variant with the exact same values for reference with the publication.

The paper also say:

> The refined formulas are named DIN99b, DIN99c, and DIN99d.

So...